### ISSUE TITLE
Where requirements go within the structure

### ISSUE THREAD

**Issue by alastc** [w3c/wcag3#155]
From an [email to the list](https://lists.w3.org/Archives/Public/w3c-wai-gl/2024OctDec/0151.html):

> Keyboard Focus Appearance doesn’t specify target size or contrast requirements. That is the same problem WCAG 2.0 has, which AGWG tried and failed to address in 2.2. This draft shows no progress in filling that gap. If anything, the lack of progress shows the new structure of WCAG 3.0 isn’t helping to address WCAG 2’s gaps and equity challenges.

There are requirements for focus-appearance (which is further along in development), however, they are further down in the structure.

This is problematic because we need _the_ or at least _a_ level of testable requirements to be part of the normative spec.  This is an issue to track progress on this topic.
